File: Reroll.Models/GameSession.cs
using System;
using System.Collections.Generic;
using System.Text;
using MongoDB.Bson;

namespace Reroll.Models
{
    public class GameSession
    {
        public ObjectId Id { get; set; }
        public string GroupName { get; set; }
        public List<Player> Players { get; set; }
        public GameMaster GameMaster { get; set; }
        public string Password { get; set; }
        public int ConnectedClients { get; set; }
    }
}

File: Reroll.Web/Reroll.Web/ClientApp/src/models/GameSession.ts

     import { Player } from './Player';
import { GameMaster } from './GameMaster';
import { ActivityMessage } from './ActivityMessage';

export class GameSession   {
	public groupName: string;
	public players: Player[];
	public gameMaster: GameMaster;
	public password: string;
	public connectedClients: number;
	public activityLogs: ActivityMessage[];
}
